lua/cfc_loadouts/client/cl_cfc_functions.lua

```diff
@@ -39,16 +54,18 @@ end
 function CFCLoadouts.loadoutFileCheck( loadoutListTable )
     for _, loadoutList in pairs( loadoutListTable ) do
         local files = file.Find( "cfc_loadout/*.json", "DATA", "dateasc" )
-        loadoutList:Clear()
+        loadoutList:Clear( )
+
         for _, filename in pairs( files ) do
             local name = string.Replace( filename, ".json", "" )
             loadoutList:AddLine( name )
         end
     end
 end
 
-function CFCLoadouts.loadoutFileCreate( fileName )
-    file.Write( "cfc_loadout/" .. fileName .. ".json", jsonTable )
+function CFCLoadouts.loadoutFileCreate( fileName, weaponsList )
+    local jsonTableSave = util.TableToJSON( weaponsList, true )
+    file.Write( "cfc_loadout/" .. fileName .. ".json", jsonTableSave )
```

Comment:
Low-value, but you could extract this to some kind of variable or convar

Reply:
As in a custom folder location?